src/form.ts

```diff
@@ -0,0 +1,162 @@
+import { Terminal } from "@tuzig/xterm"
+import { FitAddon } from "xterm-addon-fit"
+
+export type Fields = Array<{
+    prompt:string,
+    default?:string,
+    values?:Array<string>,
+    validator?:(field: string) => string,
+}>
+
+export type Results = Array<string>
+
+export function openFormsTerminal(e: HTMLElement) {
+    const terminal = new Terminal({
+        cursorBlink: true,
+        cursorStyle: "block",
+        theme: window.terminal7.conf.theme,
+        fontFamily: "FiraCode",
+        fontSize: 14,
+        rendererType: "canvas",
+        convertEol: true,
+    })
+    terminal.open(e)
+    const fitAddon = new FitAddon()
+    terminal.loadAddon(fitAddon)
+    fitAddon.fit()
+    return terminal
+}
+
+export class Form {
+
+    field: string
+    i: number
+    e: HTMLElement
+    resolve: (value) => void
+    reject: (reason?) => void
+    fields: Fields
+    results: Results
+
+    constructor(fields: Fields) {
+        this.fields = fields
+    }
+
+    chooseFields(t: Terminal) {
+        const enabled = new Array(this.fields.length).fill(false)
+        let current = 0
+        return new Promise<Array<boolean>>((resolve, reject) => {
+            t.write("\n  Choose fields to edit:")
+            t.write("\n  (Press Enter to select/deselect, D when done, Escape to cancel)")
```

Comment:
Please change to:

 "[Use arrows to move, space to select, <right> to all, <left> to none]"
 
 Enter is what you key when done, but there's no need to write it.